style.css

```diff
@@ -527,6 +571,45 @@ article h1 {
   margin-top: 25px;
 }
 
+form {
+  margin-top: -30px;
+  display: flex;
+  margin-right: 9%;
+}
+
+.footer li input {
+  border: 1px solid #cfd8dc;
+  height: 48px;
+  width: 312px;
+  margin-top: 10px;
+  border-radius: 8px;
+  padding: 15px;
+}
```

Comment:
- [ ] As one of the requirement is to parse the figma template as close as possible, please decrease the padding on your input fields to match the original design see your current design :point_down: 
![image](https://user-images.githubusercontent.com/97521557/212910511-c30565cd-1ccb-4b70-8612-9d9ffa6fb687.png)

see the figma design :point_down: 
![image](https://user-images.githubusercontent.com/97521557/212910653-b35e6858-e9c9-454a-98de-0c23a2751b64.png)